### PR TITLE
Fix for #30633 Correct possible cut and paste error.

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
Fix for #30633 Correct possible cut and paste error.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
This does look like a potential error.  However, this would need tests showing the new behavior.

**Comment by AdamSpeight2008** [comment]
@CyrusNajmabadi I don't think there needs to be a test as it still preserves existing behaviour, due the only [callsite](https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/ControlFlowAnalysis.vb#L94) preassigns the variable to false. What more interesting is why there wasn't an error for the parameter `endPointIsReachable` not being assigned in both branches because the `<Out>` attribute. Maybe the analysis went all the way back to and found the original declaration assigned as thought it was valid?

File: src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/RegionReachableWalker.vb [hunk 1]
```diff
@@ -25,7 +25,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     endPointIsReachable = If(walker._regionEndPointIsReachable.HasValue, walker._regionEndPointIsReachable.Value, walker.State.Alive)
                 Else
                     startPointIsReachable = True
-                    startPointIsReachable = False
+                    endPointIsReachable = False
```
Comment (CyrusNajmabadi) [inline src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/RegionReachableWalker.vb]: actually, it looks like the caller sets both of these to true if there is a failure.  so perhaps this should be `endPointIsReachable = true`?

**Comment by jaredpar** [comment]
>  I don't think there needs to be a test as it still preserves existing behaviour,

If the change isn't worth a test then it won't be considered for merging.

**Comment by jaredpar** [comment]
Closing as the PR is getting stale and issues haven't been addressed.

**Pull request closed without merging**